config/enabled.yml

```diff
@@ -1395,6 +1395,10 @@ Lint/UnneededDisable:
                  It must be explicitly disabled.
   Enabled: true
 
+Lint/UnneededRequireStatement:
+  Description: 'Checks for unnecessary require statement.'
```

Comment:
[IMO] "require" is a part of Ruby program. So, I think we should surround this with backtick.